SweetTweetApp/src/main/java/JSONDecorator/JSONDecorator.java

```diff
@@ -1,14 +1,14 @@
-package servletsForPosts;
+package JSONDecorator;
 
 import com.google.gson.Gson;
 import com.google.gson.GsonBuilder;
 import com.google.gson.JsonElement;
 import net.dongliu.gson.GsonJava8TypeAdapterFactory;
 
 public class JSONDecorator {
-    static Gson gson;
+    public static Gson gson;
 
-    JSONDecorator() {
+    public JSONDecorator() {
```

Comment:
It seems to has been told several times to avoid two very bad things
1) declaring public static non-final fields  (these must be `public static final Something CONSTANT` if absoultely must)
2) initializing static fields in an instance context
Technically, I may have written this in someone else's PR but I anyway seem to have told this in public... Anyeay, don't just do this)